python/paddle/fluid/framework.py

```diff
@@ -154,6 +154,7 @@ def __init__(self,
                  shape=None,
                  dtype=None,
                  lod_level=None,
+                 capacity=None,
```

Comment:
We need to document what is capacity.

Reply:
Thank you for pointing this out. Updated the comments.